cuegui/cuegui/config/cuegui.yaml

```diff
@@ -64,6 +64,10 @@ editor.windows: 'notepad'
 editor.mac: 'open -t'
 editor.linux: 'gview -R -m -M -U {config_path}/gvimrc +'
 
+viewer.windows: 'rv'
+viewer.mac: 'rv'
+viewer.linux: 'rv'
```

Comment:
Please add a comment explaining what this property means

Reply:
This had been added now